incubator/kube-backup/templates/NOTES.txt

```diff
@@ -0,0 +1,85 @@
+1) To make backup working you need to have s3 bucket created.
```

Comment:
Use terraform comments (`#`) - that way the entire thing can be copy & pasted into a `.tf` file.